src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -47,6 +47,9 @@ public function getHelperSet()
      */
     public static function strlen($string)
     {
+        if ( ! \is_string($string)) {
+            $string = (string)$string;
+        }
```

Comment:
```suggestion
        $string = (string) $string;
```

There are more places than this. We don't need to type-check here because a string cast on a string should be a no-op anyway.